src/App.js

```diff
@@ -10,6 +10,11 @@ if (process.env.REACT_APP_MOCK_API_TRUE) {
 }
 
 function App() {
+  useEffect(() => {
+    ReactGa.initialize("UA-180314898-1");
```

Comment:
put the TRACKING_ID in constants file

Reply:
this will not run because eslint will throw a compile error
need to do an import like below
import React, { useEffect } from 'react'
import { initialize, pageview } from 'react-ga'

![image](https://user-images.githubusercontent.com/42846522/96355147-69c6fd00-10ac-11eb-842b-a01b32b761e8.png)

Reply:
good points. Since I cloned again to a different folder to test why I couldn't push I didn’t realize I pushed from the new repo with no changes and I had to copy stuff over and missed the imports. My bad. 
I don't believe we need to specifically import initialize or pageview from within react-ga because I could use them that way but I changed it to your suggestion as that looks cleaner. Thank you!

Reply:
Thanks for changing it according to my suggestions. In regards to initialize and pageview, you had to import it because eslint will not let you run it without importing it. Please see screen capture I pasted above. You are right that we could've used it, but we are using eslint to help improve our code. In our case, this is required. :-)